### DISCUSSION TITLE
mcVersionRp not working/reliable.

### DISCUSSION THREAD

**Discussion by justinrest** [HypixelDev/PublicAPI#502] [General]
For a user's mcVersionRp value in the API sometimes it doesn't exist and sometimes mostRecentMinecraftVersion has no data either.

View this forum post here: https://hypixel.net/threads/api-api-doesnt-store-version-for-all-players.3494233/

**Comment by mdashlw** [comment]
It has never been reliable. I think it should be removed, if not changed, because some people think it is reliable and make wrong decisions/conclusions.

**Comment by justinrest** [comment]
> It has never been reliable. I think it should be removed, if not changed, because some people think it is reliable and make wrong decisions/conclusions.

agree
